api/src/main/java/pl/kolendateam/dadcard/feats/entity/CharacterFeat.java

```diff
@@ -123,5 +123,189 @@ public boolean checkPrerequisite(
         return checkEvry==ckeckTot;
 
     }
+
+    public String wildShape(int levelOfFeat){
+        String stringFeatSpecial = "";
+
+        if(levelOfFeat < 4){
+            stringFeatSpecial = levelOfFeat + "/day";
+        }
+        if(levelOfFeat == 4){
+            stringFeatSpecial = "3/day (Large)";
+        }
+        if(levelOfFeat == 5){
+            stringFeatSpecial = "4/day (Large)";
+        }
+        if(levelOfFeat == 6){
+            stringFeatSpecial = "4/day (Large and Tiny)";
+        }
+        if(levelOfFeat == 7){
+            stringFeatSpecial = "4/day (Large, Tiny and plant)";
+        }
+        if(levelOfFeat == 8){
+            stringFeatSpecial = "5/day (Large, Tiny and plant)";
+        }
+        if(levelOfFeat == 9){
+            stringFeatSpecial = "5/day (Large, Tiny, Huge and plant)";
+        }
+        if(levelOfFeat == 10){
+            stringFeatSpecial = "5/day (Large, Tiny, Huge and plant), 1/day (elemental)";
+        }
+        if(levelOfFeat == 11){
+            stringFeatSpecial = "6/day (Large, Tiny, Huge and plant), 2/day (elemental)";
+        }
+        if(levelOfFeat == 12){
+            stringFeatSpecial = "6/day (Large, Tiny, Huge and plant), 3/day (Huge elemental)";
+        }
+        return stringFeatSpecial;
+    }
+
+    public String flurryOfBlows(int levelOfFeat) {
+        String stringFeatSpecial = switch(levelOfFeat) {
+            case 1 -> "-2/-2";
```

Comment:
co robi ->? switch cos zwraca? Można to tez ograc tak 
case 1:
return "-2/2";

Reply:
to taka nowinka Operator strzałki (->) https://javastart.pl/baza-wiedzy/java-podstawy-jezyka/switch-expressions